nucypher/blockchain/eth/deployers.py

```diff
@@ -400,9 +400,12 @@ def deploy(self,
         if deployment_mode != FULL:
             raise self.ContractDeploymentError(f"{self.contract_name} cannot be deployed in {deployment_mode} mode")
 
-        self.check_deployment_readiness()
+        self.check_deployment_readiness(ignore_deployed=ignore_deployed)
+        
+        if emitter:
+            emitter.message("\nNext Transaction: Token Contract Creation", color='blue', bold=True)
 
-        # Order-sensitive!
+        # WARNING: Order-sensitive!
         constructor_kwargs = {"_totalSupplyOfTokens": self.economics.erc20_total_supply}
         constructor_kwargs.update(overrides)
```

Comment:
Curious for what line this comment (order-sensitive)?

Reply:
Hmm - I'm not 100% sure what the comment is referring to, however we are mutating a constructor dictionary here with `.update`  *then* performing filtration, so my sense is that it is related to these three lines being called in this order.

Reply:
Got it, first my thought was that comment related to dict not to order of lines